internal/provider/gql.go

```diff
@@ -127,6 +127,22 @@ type orgPayload struct {
 	Mrn         string
 	Name        string
 	Description string
+	SpacesCount int
+	SpacesList  struct {
```

Comment:
I think this does not handle pagination properly

Reply:
Right. Let me handle pagination here to fetch all spaces at once.